multichain-testing/scripts/ymax-tool.ts

```diff
@@ -434,21 +432,25 @@ const main = async (
     const { [contract]: instance } = fromEntries(
       await walletKit.readPublished('agoricNames.instance'),
     );
-    const result = await walletStore.saveOfferResult(
+    const meta = await walletStore.saveOfferResult(
       { instance, description },
       description.replace(/^deliver /, ''),
     );
-    trace('redeem result', result);
+    trace('redeem result', meta.result);
     return;
   }
 
-  const yc = walletStore.get<ContractControl<YMaxStartFn>>('ymaxControl');
-
   if (values.getCreatorFacet) {
-    await walletStore.savingResult('creatorFacet', () => yc.getCreatorFacet());
+    const yc =
+      walletStore.getForSavingResults<ContractControl<YMaxStartFn>>(
+        'ymaxControl',
+      );
+    await yc.getCreatorFacet({ name: 'creatorFacet' });
```

Comment:
If this isn't sufficiently clear, we could consider renaming `name` to something that more obviously indicates that it specifies the name in the wallet store at which method results should be saved.